### PR TITLE
Update dotnet-dump docs

### PR DESCRIPTION
**Pull request by Youssef1313**

### CODE REVIEW

File: docs/core/diagnostics/dotnet-dump.md
Comment (Youssef1313) [inline docs/core/diagnostics/dotnet-dump.md:224]: This PR should wait until #19697 is merged.

**Review by sdmaclea — COMMENTED** [review]
LGTM

**Review by tdykstra — APPROVED** [review]
Thanks @Youssef1313 !

**Pull request merged**
